### PR TITLE
feat: gura 3 mil project

### PR DESCRIPTION
**Pull request by Edqe14**

### CODE REVIEW

File: pages/projects/[id].tsx [hunk 1]
```diff
@@ -117,13 +118,13 @@ export default function ProjectPage({ doc, allSubmissions }: IProps) {
 	}
 
 	let themeStyle = 'theme-ina';
-	if (router.query.id === '3') {
+	if (router.query.id === '3' || router.query.id === '6') {
 		themeStyle = 'theme-gura';
 	}
```
Comment (GoldElysium) [inline pages/projects/[id].tsx:124]:
```suggestion
if(doc.flags?.includes('gura3mil')) {
	return <Phaser scene="gura3mil" data={{ submissions: allSubmissions }} />;
}
```

**Review by GoldElysium — CHANGES_REQUESTED** [review]

File: components/phaser/gura3mil/plugins/ui.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+	}
+
+	ensureOrientation() {
+		try {
+			// Watch for fullscreen and always lock to landscape
+			this.game.scale.on('enterfullscreen', () => this.lock()).on('leavefullscreen', () => this.unlock());
+			// Force fullscreen
+			this.game.scale.startFullscreen();
+		} catch {
+			console.warn('Cannot lock orientation');
```
Comment (GoldElysium) [inline components/phaser/gura3mil/plugins/ui.tsx:45]:
```suggestion
			// eslint-disable-next-line no-console
			console.warn('Cannot lock orientation');
```

File: components/project/Submissions.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+import InfiniteScroll from 'react-infinite-scroll-component';
+import TextHeader from '../TextHeader';
+import { ISubmission } from '../../models/Submission';
+
+interface ISub {
+	data: ISubmission,
+	index: number
+}
+
+const Submission = ({ data, index }: ISub) => (
```
Comment (GoldElysium) [inline components/project/Submissions.tsx:12]:
```suggestion
interface ISubmissionProps {
	data: ISubmission,
	index: number
}

const Submission = ({ data, index }: ISubmissionProps) => (
```
For naming consistency

**Review by GoldElysium — CHANGES_REQUESTED** [review]

File: components/phaser/gura3mil/main.tsx [hunk 4]
```diff
@@ -99,7 +99,15 @@ class Main extends Phaser.Scene {
 			.setAlpha(0)
 			.on('scroll', ({ t }: { t: number }) => {
 				if (t >= 0.8) {
-					if (submissions.length <= 0) return this.panel.removeAllListeners('scroll');
+					if (submissions.length <= 0) {
```
Comment (GoldElysium) [inline components/phaser/gura3mil/main.tsx:102]: Don't get why it's `<= 0` instead of `=== 0` since it's never below 0, but whatever.

**Review by GoldElysium — DISMISSED** [review]
As of now, don't see any issues with it.

File: components/phaser/gura3mil/main.tsx (hunk 4 above)
Comment (Edqe14) [reply components/phaser/gura3mil/main.tsx:137]: I did it to be "safe" but yeah

**Review by GoldElysium — APPROVED** [review]
image

**Review by GoldElysium — APPROVED** [review]
image
Now actually done

**Review by GoldElysium — APPROVED** [review]

**Pull request merged**